securedrop/request_that_secures_file_uploads.py

```diff
@@ -24,7 +24,7 @@ def _secure_file_stream(self, total_content_length, content_type,
             # note in `config.py` for more info. Instead, we just use
             # `/tmp`, which has the additional benefit of being
             # automatically cleared on reboot.
-            return SecureTemporaryFile('/tmp')
+            return SecureTemporaryFile('/tmp')  # nosec
```

Comment:
Why does this need to be marked `# nosec`?

Reply:
There is a rule in bandit that is a string match for /tmp `/tmp`:
```
>> Issue: [B108:hardcoded_tmp_directory] Probable insecure usage of temp file/directory.
   Severity: Medium   Confidence: Medium
   Location: ./securedrop/request_that_secures_file_uploads.py:27
```
Which warns against the usage of /tmp to store application data (https://www.owasp.org/index.php/Insecure_Temporary_File). Given the way `filesystem_id` is generated and files are encrypted as they are streamed to disk, it shouldn't affect the confidentiality of the submission. As for integrity, perhaps using an authenticated encryption mode (GCM) *might* be an improvement, but would obviously require very careful consideration.

Reply:
Could you open a ticket for this? Not necessarily for an AEAD, but for maybe moving it to `/var/run/securedrop/tmp`.

Reply:
https://github.com/freedomofpress/securedrop/issues/3067